### ISSUE TITLE
Parsing error: The keyword 'export' is reserved

### ISSUE THREAD

**Issue by bytrangle** [9gustin/react-notion-render#163]
That's the error I got from Eslint whenever the keyword "export" or "import" is used.

I've tried restarting my code editor (VSCode), to adding parseOptions to Eslint Config, to no avail:
 
```
{
  "env": {
    "jest": true,
    "es6": true
  },
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    },
    "ecmaVersion": 2018,
    "sourceType": "module"
  }
}
```

The only thing I haven't tried is to use `babel-eslint` as a parser 😅. What do you recommend?

**Comment by 9gustin** [comment]
When you have this error? running what command? Maybe the linter was checking my package and you can ignore the folder node_modules?

**Comment by bytrangle** [comment]
You didn't have this error? I just opened any `.js` file, for example `dev-example/pages/index.js` and got the error. Opening `.tsx` files didn't cause the error.

**Comment by 9gustin** [comment]
Thanks for solving that, i really appreciate it 💖 i have removed eslint because i wanted to configure it again and i dont notice that, solved in v3.10.1

Install it with:
`npm i @9gustin/react-notion-render@latest`
